Storage/Storage/CoreData/CoreDataManager.swift

```diff
@@ -48,34 +48,22 @@ public class CoreDataManager: StorageManagerType {
 
             /// Backup the old Store
             ///
+            var persistentStoreBackupError: Error?
             do {
                 let sourceURL = self.storeURL
                 let backupURL = sourceURL.appendingPathExtension("~")
                 try FileManager.default.copyItem(at: sourceURL, to: backupURL)
             } catch {
-                let message = "☠️ [CoreDataManager] Cannot backup Store: \(error)"
-                self.crashLogger.logMessageAndWait(message,
-                                                   properties: ["persistentStoreLoadingError": persistentStoreLoadingError,
-                                                                "backupError": error,
-                                                                "appState": UIApplication.shared.applicationState.rawValue,
-                                                                "migrationMessages": migrationDebugMessages],
-                                                   level: .fatal)
-                fatalError(message)
+                persistentStoreBackupError = error
             }
 
             /// Remove the old Store
             ///
+            var persistentStoreRemovalError: Error?
             do {
                 try FileManager.default.removeItem(at: self.storeURL)
```

Comment:
This one might actually be wrong because it does not consider the existence of the `.sqlite-wal` and `.sqlite-shm` files. 

![image](https://user-images.githubusercontent.com/198826/86027013-61f42780-b9ed-11ea-9ab8-4f38918deaac.png)

Reply:
yea they mentioned this at the WWDC lab. I believe https://github.com/woocommerce/woocommerce-ios/issues/2473 is going to replace this with [`NSPersistentStoreCoordinator`'s `destroyPersistentStore`](https://developer.apple.com/documentation/coredata/nspersistentstorecoordinator/1468888-destroypersistentstore)